server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/BackupAction.java

```diff
@@ -31,45 +30,49 @@
 import org.sonar.db.DbSession;
 import org.sonar.db.qualityprofile.QualityProfileDto;
 import org.sonar.server.qualityprofile.QProfileBackuper;
-import org.sonar.server.qualityprofile.QProfileFactory;
-import org.sonar.server.qualityprofile.QProfileRef;
 import org.sonarqube.ws.MediaTypes;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
+
 public class BackupAction implements QProfileWsAction {
 
   private final DbClient dbClient;
   private final QProfileBackuper backuper;
-  private final QProfileFactory profileFactory;
+  private final QProfileWsSupport wsSupport;
   private final Languages languages;
 
-  public BackupAction(DbClient dbClient, QProfileBackuper backuper, QProfileFactory profileFactory, Languages languages) {
+  public BackupAction(DbClient dbClient, QProfileBackuper backuper, QProfileWsSupport wsSupport, Languages languages) {
     this.dbClient = dbClient;
     this.backuper = backuper;
-    this.profileFactory = profileFactory;
+    this.wsSupport = wsSupport;
     this.languages = languages;
   }
 
   @Override
   public void define(WebService.NewController controller) {
-    NewAction backup = controller.createAction("backup")
+    NewAction action = controller.createAction("backup")
       .setSince("5.2")
       .setDescription("Backup a quality profile in XML form. The exported profile can be restored through api/qualityprofiles/restore.")
       .setResponseExample(getClass().getResource("backup-example.xml"))
       .setHandler(this);
 
-    QProfileRef.defineParams(backup, languages);
+    QProfileReference.defineParams(action, languages);
+
+    wsSupport.createOrganizationParam(action)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2209)